tools/rust/cargo-proton-bundle/src/bundle/msi_bundle.rs

```diff
@@ -1,36 +1,12 @@
+#[warn(dead_code)]
```

Comment:
that's nice

Reply:
Yeah, it can be removed but I put it in there for my own sanity.